### ISSUE TITLE
battery descriptions

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#77]
@terracoda in the design document for #74, sometimes the battery descriptions should be

"More than B Batteries".

and sometimes

"Between X and Y Batteries."

There is a note in the document that says "Through play students learn that "more than" means a partial battery."

Do we still need "Between..." description?  If so, when should it be the description over "More than..."?

**Comment by terracoda** [comment]
Ooops, let me review that content.

**Comment by terracoda** [comment]
@jessegreenberg, sorry for the late response. I think people preferred the "between x and y batteries" phrasing. Let's go with that.

**Comment by terracoda** [comment]
@jessegreenberg, I updated the design spreadsheet to reflect this decision.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, changed in the above commit.  Closing.
